Source/WebKit/GPUProcess/media/RemoteMediaPlayerProxy.cpp

```diff
@@ -957,8 +957,10 @@ void RemoteMediaPlayerProxy::updateCachedState(bool forceCurrentTimeUpdate)
     maybeUpdateCachedVideoMetrics();
     if (m_bufferedChanged) {
         m_bufferedChanged = false;
-        if (m_engineIdentifier != MediaPlayerEnums::MediaEngineIdentifier::AVFoundationMSE)
+        if ((m_engineIdentifier != MediaPlayerEnums::MediaEngineIdentifier::AVFoundationMSE)
+            && (m_engineIdentifier != MediaPlayerEnums::MediaEngineIdentifier::MockMSE)) {
```

Comment:
No need for two ()

Reply:
that's not what the style checker think. As soon as you have && or || , it requires all two expression to be within ()